CHARTER.md

```diff
@@ -1,32 +1,35 @@
-# {{GROUP_NAME}} Charter
-<!--
- Provide an introduction summarising the goals and motivation behind your
- project group.
--->
+# Portable SIMD Charter
+
+Build and stabilize a portable SIMD API to the standard library under `core::simd` and `std::simd`.
+Draw elements from the existing portable SIMD implementations out in the ecosystem:
+
+- [`generic-simd`]
+- [`packed_simd`]
+- [`wide`]
 
 ## Goals
 
-<!--
- Explain what changes you'd like to see your group your group to focus on, and
- how you plan to approach these issues. Focus on explaining the highest possible
- level of your change.
--->
+- Determine the shape of the portable SIMD API.
+- Get an unstable `std::simd` and `core::simd` API in the standard library.
+This may mean renaming `packed_simd` to `stdsimd` and working directly on it, or creating a new repository and pulling in chunks of code as needed.
+- Produce a stabilization plan to allow portions of the API to be stabilized when they're ready, and coordinate with other unstable features.
+- Respond to user feedback and review contributions to the API.
+- Update [RFC 2948] based on the final API and stabilization plan.
+- Stabilize!
 
 ## Constraints And Considerations
 
-<!--
- Explain the scope of your group, what you have chosen not to include in
- your goals, and your motivations behind making them non-goals.
--->
-
+The initial implementation will be built on LLVM intrinsics directly rather than `core::arch`.
+This is so we don't have to block the portable API on filling in a lot of missing intrinsics in `core::arch` on non-x86 platforms.
+Migrating from LLVM intrinsics to `core::arch` is kept as a future possibility.
```

Comment:
I think this would also require some sort of detection of `#[target_feature]`, since LLVM intrinsics already interact with it correctly.  Not sure if it's worth noting

Reply:
As far as I'm aware, you tell LLVM what you want and you tell it the features available and it solves the problem as best it can. Which is what makes intrinsics so powerful.

Reply:
Sorry--I meant that switching to `core::arch` would probably require a language change as well, to allow that same interaction.

Reply:
Ah, yes. Moving to a `core::arch` based implementation would be a Tall Task, to say the least.

Reply:
I guess what I was trying to get at with this was consciously not painting ourselves into any corners that would make moving to `core::arch` totally infeasible instead of just lots of work. It probably goes without saying. Should I reword it or just cut it out?

Reply:
I agree that we should make it clear it's not _exclusively_ possible with LLVM intrinsics, but I think this is potentially misleading when the language feature is probably a bigger hurdle than expanding `core::arch`.  Not sure the best way of phrasing that, though!  For reference, https://github.com/rust-lang/rust/issues/42515 seems to be the associated issue.

Reply:
It sounds like we’ll want to capture this, but probably not in the charter 🙂 I’ll just remove this note :+1: